helm/release-exporter/templates/deployment.yaml

```diff
@@ -18,20 +18,19 @@ spec:
       containers:
       - name: release-exporter
         image: quay.io/giantswarm/release-exporter:{{ .Chart.Version }}
+        args:
+        - --run-every=30
```

Comment:
Is that every 30 seconds? That's a bit too much right?

Reply:
it is every 30 min. I guess it is better to use k8s notation adding `s`, `m` or `h` right?

Reply:
It would make it easier to read yes :D

Reply:
done